proto/sdk/alpha/alpha.proto

```diff
@@ -146,4 +169,48 @@ message PlayerID {
 // List of Player IDs
 message PlayerIDList {
     repeated string list = 1;
-}
\ No newline at end of file
+}
+
+// A representation of a Counter.
+message Counter {
```

Comment:
I will update to have the Name in the Counter message per the RPC guidelines. Doesn't look like it will be an issue as we have our own custom converter for SDK -> GameServer in /agones/pkg/sdkserver/sdk.go.